### DISCUSSION TITLE
Good Linux Laptop/Hardware with decent Pen Input?

### DISCUSSION THREAD

**Discussion by lstwn** [flxzt/rnote#77] [Q&A]
Hi,

I'd love to use something like rnote on Linux, mostly for annotating university papers, lectures and so on.
As great as rnote is already looking (but quite useless without good pen support), I'm desperate for good hardware recommendations. Unfortunately, my search for a decent Linux compatible laptop/convertible with pen input was quite unsuccessful because there aren't many reviews (or rather official remarks from the manufacturers) to rely on.

Hence, I might ask here: With what hardware/input do you use rnote?

**Comment by gloopsies** [comment]
I personally use my desktop with One by Wacom tablet plugged in. I sometimes use the same tablet with my laptop but it is not as easy to use as a 2 in 1 device. I don't know any specific 2 in 1 device that is sold with Linux preinstalled but at least where I come from most laptops come without an OS or there is a more expensive Windows version

**Comment by flxzt** [comment]
I can only speak for what works for me, but I have a good experience with the lenovo thinkpad x1 yoga. I think lenovo is one of the few vendors that give the option to ship with linux preinstalled ([look here](https://support.lenovo.com/de/de/solutions/pd031426-linux-for-personal-systems)), so their hardware naturally works pretty good. (But the thinkpads tend to be expensive, so look out for student deals. I got mine for I think around 30% cheaper than retail.)

**Reply by defini-tiv** [reply]
Which generation do you have?
I have a gen.3 and i am really happy with it. Except the touchscreen sometimes isn't usable after suspend, do you have any experience with this?

**Reply by jenskuespert** [reply]
Maybe off-topic - but here it goes...

I had the same problem here. Solved it with the following information:

https://wiki.archlinux.org/title/Lenovo_ThinkPad_X1_Yoga_(Gen_3)

Good luck!

**Comment by lazz0** [comment]
Hello lstwn,

I have been using a Microsoft Surface Pro 3 with a surface pen for quite some time now with ubuntu and it has been working pretty good for me so far. You might need to install the linux-surface kernel for better support. That is of course only my personal experience.
It is not the most up-to-date device in terms of hardware, but it has definitely suited my needs fairly well (heavy use of office software, browsing with multiple open tabs, image editing with gimp, presentations in professional settings, etc.). The only issue  I got is that it sometimes can get quite hot.
I bought it second hand for a pretty ok price in comparison to other (especially new) 2-in-1/touch screen devices. Getting a cheaper device at first might be a good way to dip your toes.
From what i read linux should be reasonable usable with a stylus on Surface devices up to the SP6, although I have no first hand experience with that. You can find an overview of supported features in this chart: https://github.com/linux-surface/linux-surface/wiki/Supported-Devices-and-Features#feature-matrix

I hope you find a device that suits your needs. I would be interested to know what worked (or hasn't worked) for you.

**Reply by lstwn** [reply]
Thanks for your inputs, I really appreciate it :)

"Unfortunately", I've settled for an iPad for my study notes because I've quite successfully locked myself into the Apple ecosystem due to my work issued MacBook and iPhone.. Maybe I'm reconsidering this choice with my next personal machine because I miss the freedom of Linux!

**Comment by StrandedAlien** [comment]
Hi, you guys might want to try a Distro with Gnome Desktop. I use Manjaro with Gnome and have found everything concerning the stylus fully functioning on a HP X360 convertible with original HP pressure sensitive pen. These thingies come quite cheap on Ebay right now, coz many companies replace them. They have a Gorilla glass screen so no trouble with scratches. I can leave the hand on the screen and draw with the pen, switch auto-rotate on off in tablet mode, use onscreen keyboard... All out of the box. I am impressed by Gnome stylus andManjaros X360 support. Some of the devs seem to own the same hardware. I can really recommend this setup.
